src/services/api-claim-service.ts

```diff
@@ -24,11 +27,30 @@ export class ApiClaimService {
     }
 
     this.addApiClaim(mfeName, apiClaim)
-
     this.updateMfeConfigApi(mfeName, apiClaim.name, serviceUrl)
   }
 
-  private addApiClaim(mfeName: string, apiClaim: ApiClaim): void {
+  public addExternalApiClaim(
+    mfeName: string,
+    apiClaim: ExternalApiClaim
+  ): void {
+    const url = this.cmService.getBundleMicroserviceUrl(
+      apiClaim.bundleId,
+      apiClaim.serviceId
+    )
+
+    if (!url) {
+      throw new CLIError('Failed to get microservice URL')
+    }
+
+    this.addApiClaim(mfeName, apiClaim)
```

Comment:
Warning, we are adding an external API claim, so you don't need to check if the referenced MS is contained into the bundle (it's an external one, so the validation is done on CM side)

Reply:
Ah good one. Didn't notice that.